src/main/java/org/broadinstitute/hellbender/engine/FeatureWalker.java

```diff
@@ -40,9 +39,9 @@ private void initializeDrivingFeatures() {
         final File drivingFile = getDrivingFeatureFile();
         final FeatureCodec<? extends Feature, ?> codec = FeatureManager.getCodecForFile(drivingFile);
         if (isAcceptableFeatureType(codec.getFeatureType())) {
-            drivingFeatures = new FeatureDataSource<>(drivingFile, (FeatureCodec<F, ?>)codec);
+            drivingFeatures = new FeatureDataSource<>(drivingFile);
 
-            final FeatureInput<F> drivingFeaturesInput = new FeatureInput<>("drivingFeatureFile", Collections.emptyMap(), drivingFile);
+            final FeatureInput<F> drivingFeaturesInput = new FeatureInput<>(drivingFile.getAbsolutePath(), "drivingFeatureFile");
```

Comment:
Reversing the order of consecutive String arguments makes me nervous, but ok provided you've audited all usages for potential logical name <-> path swaps.